lib/containers/CreateNew.js

```diff
@@ -21,20 +31,56 @@ class CreateNew extends PureComponent {
     return {};
   }
 
+  async select(key, value) {
+    await this.setState(
```

Comment:
why the extra lines of code for the Object.assign? Isn't `setState` handling this to make sure there are no inadvertent mutations? Also, if it is necessary, might be cleaner with a spread: `this.setState({ ...this.state, [key]: value })`, unless... does that cause problems with the variable as key? Last question, why `await` here? Apparently `setState` is async for performance reasons (Dan Abrimov has an explanation [here](https://github.com/facebook/react/issues/11527#issuecomment-360199710)). `setState` can accept a callback as a second arg if that's necessary, but also found some interesting options for making an `await`able version of `setState`, e.g. https://medium.com/front-end-hacking/async-await-with-react-lifecycle-methods-802e7760d802